docs/develop/05_dApp/02_well-known-did-config.md

```diff
@@ -0,0 +1,149 @@
+---
+id: well-known-did-config
+title: Well Known DID Configuration
+---
+
+The Well Known DID Configuration is implemented as a security measure when setting up the communication session between dApp and extension.
+
+The implementation is based on the [Well Known DID Configuration](https://identity.foundation/.well-known/resources/did-configuration/) specified by the Decentralized Identity Foundation:
+
+## dApp Steps
+
+Add the following URI to your dApp's root. The extension will make an HTTP GET request to this URI, and your dApp will respond with the domain configuration resource.
+
+`/.well-known/did-configuration.json`
+
+### Attesting the Domain Linkage Credential
+
+Upon receiving the GET request from the extension, your dApp will make a claim that contains its domain origin and DID. Then it will self-attest the claim. This self-attestation is referred to as a Domain Linkage Credential.
```

Comment:
is this done ad-hoc, or do I have to prepare it beforehand?

Reply:
SKYC does it ad-hoc

Reply:
including self-attesting and putting it on-chain? Do I need to wait at least 12 seconds before I can connect to SKYC?

Reply:
It's not on-chain

Reply:
Got it, thanks